pkgs/by-name/ec/ec2-instance-selector/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "ec2-instance-selector";
+  version = "3.1.1";
+
+  src = fetchFromGitHub {
+    owner = "aws";
+    repo = "amazon-${pname}";
+    tag = "v${version}";
```

using finalAttrs in this case would be better
```suggestion
buildGoModule (finalAttrs: {
  pname = "ec2-instance-selector";
  version = "3.1.1";

  src = fetchFromGitHub {
    owner = "aws";
    repo = "amazon-${pname}";
    tag = "v${finalAttrs.version}";
```